### PR TITLE
Anonymous users in OIDC module

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/appstore/modules/oidc.md [hunk 1]
```diff
@@ -164,6 +164,8 @@ You may have to add the *Anonymous* user role if it does not exist already.
 
 ### 4.2 Allowing Anonymous Users
 
+The OIDC module supports multiple OIDC/OAuth-compatible IdPs. If you want to allow your end users to choose from a number of different IdPs, you will need to give them access to the app to choose their preferred IdP. Because they need to do this before they have signed in to the app, you need to give anonymous users access to your app. If you have only configured a single IdP, you do not need to allow anonymous users.
+
```
Comment (SharishmaPodduturi999) [inline content/en/docs/appstore/modules/oidc.md:168]: Hi Mark,
I have checked the documentation of OIDC SSO module.The missed part in the docs is [Navigation](https://docs.mendix.com/appstore/modules/oidc/#43-configuring-navigation)
Add Role-based home page(Login_Web_Button page) for Anonymous userroles in the Navigation
Screenshot 2023-03-06 at 6 46 07 PM

To be clear:
If we enable/disable anonymous users, users can configure multiple/single IDPs for their app but they will not be able to see the configured IDPs when they logout if they don't configure the Login_Web_button page in the Navigation
Screenshot 2023-03-10 at 9 51 12 AM

Even though if they configure single IDP, they need to allow anonymous users.

Can you please remove the below line
`If you have only configured a single IdP, you do not need to allow anonymous users.`

and add the missing part in the navigation

Comment (MarkvanMents) [reply content/en/docs/appstore/modules/oidc.md:168]: @SharishmaPodduturi999 
I'm puzzled that this is not the experience of the customer who raised the issue of anonymous users #5706. They say that it works fine for them without enabling anonymous users.
However, I'll make the changes you suggest for now and report back to the customer once they have been made.

Comment (MarkvanMents) [reply content/en/docs/appstore/modules/oidc.md:168]: I've updated this in https://github.com/mendix/docs/pull/5750/commits/87f61f397416e1d56bb77a2e2828657a5f164bc6.
Does it look OK now?

Comment (SharishmaPodduturi999) [reply content/en/docs/appstore/modules/oidc.md:168]: @MarkvanMents It looks good to me

Comment (MarkvanMents) [reply content/en/docs/appstore/modules/oidc.md:168]: Thanks - I'll publish now.

**Pull request merged**